HW02/bright_okike/hw02.rb

```diff
@@ -0,0 +1,70 @@
+module My_enumerables
+  def my_each
+    self.length.times do |x|
+      yield(self[x])
+    end
+  end
+
+  def my_map(proc = nil)
+    arr = Array.new
+    if proc 
+      self.length.times do |x|
+        arr.push(proc.call(self[x]))
+      end
+    else 
+      self.length.times do |x|
+        arr.push(yield(self[x]))
+      end
+    end
+    arr
+  end
+
+  def my_select
+    arr = Array.new
+    self.length.times do |x|
+      if yield(self[x])
+        arr.push(self[x])
+      end
+    end
+    arr
+  end
+
+  def my_all?
+    self.length.times do |x|
+        !yield(self[x]) ? false : true
+    end
+  end
+
+  def my_any?
+    self.length.times do |x|
+      return true if yield(self[x]) != false
+    end
+    false
+  end
+
+  def my_none?
+    self.length.times do |x|
+      return false if yield(self[x]) 
+    end
+    true
+  end
+
+  def my_count arg=nil
```

Comment:
```suggestion
  def my_count(arg = nil)
```
Write in the same style as in line 8